bolt-design-system/bolt
# Feature/bds 761 list component refactoring

## Jira

http://vjira2:8080/browse/BDS-761

## Summary

Creating Web Component version of list component.

## Details

Retype JS and SCSS to work with Web Components 

## How to test

Run the branch locally and check pages under Components->List in the navigation.
